hugo/data/teams.yml

```diff
@@ -37,7 +37,7 @@ sections:
           name: David Pordomingo
           position: Software Engineer
           github: https://github.com/dpordomingo
-        - picture: placeholder/male.png
+        - picture: lou.png
```

Comment:
If you rename it to "marvin.png" it's better 😋

Reply:
Done.

Reply:
Thanks! Anyway I guess that also here in the `yml` is to be changed.